### ISSUE TITLE
ActiveModel::Errors#keys is deprecated

### ISSUE THREAD

**Issue by thisismydesign** [jsonapi-rb/jsonapi-rails#127]
`render jsonapi_errors: entity.errors`

```
DEPRECATION WARNING: ActiveModel::Errors#keys is deprecated and will be removed in Rails 6.2.

To achieve the same use:

  errors.attribute_names
```

**Comment by tagliala** [comment]
This has been fixed via #130 and it is not an issue on `master` branch.

Any chance to release a new version?

**Comment by remear** [comment]
I'll close this out. A new version is coming, but I can't say when. I'm still reviewing what we can merge for this release, and working to fix the CI.
